packages/babel-parser/test/fixtures/comments/basic/async-arrow-function/input.js

```diff
@@ -0,0 +1,3 @@
+/* 1 */ async /* 2 */ (/* 3 */) /* 4 */=>x;
```

Comment:
The current main fails to attach `/* 2 */` and `/* 3 */`, and incorrectly attaches `/* 4 */` to leading comments of `x`.